internal/webserver/handler-api.go

```diff
@@ -434,12 +434,32 @@ func (h *Handler) ApiDownloadEbook(w http.ResponseWriter, r *http.Request, ps ht
 				ContentType: contentType,
 			}
 
-			book, err = core.GenerateEbook(request)
-			content.Close()
+			// if file exist book return avilable file
+			strID := strconv.Itoa(book.ID)
+			ebookPath := fp.Join(request.DataDir, "ebook", strID)
```

Comment:
Here you are not using the `${strID}.epub` filename. We need to be consistent. Maybe creating a function called `GetEbookFilename(book model.Bookmark) string` that receives the bookmark and returns the ebook filename?